multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -140,6 +142,9 @@ def __init__(
             (options: 'binary', 'multiclass', 'regression').
             If `problem_type = None`, the prediction problem type is inferred
             based on the label-values in provided dataset.
+        pipeline
+            This defines inference tasks like FeatureExtraction, ZeroShotClassification, etc.
```

Comment:
Are these all inference tasks? Or there are training tasks?

Reply:
Currently we only use this for inference as what huggingface does: (https://huggingface.co/docs/transformers/main_classes/pipelines). But if later we find this flag useful in training we may change this.